README.md

```diff
@@ -1,2 +1,60 @@
-# react-calculator
-Simple calculator web app created using React
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
+Simple calculator web app built using React
+
+
+## Built With
+
+- HTML/CSS/JS
+- React
+
+## Live Demo (if available)
+
+[Live Demo Link] (COMING SOON)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+(COMING SOON)
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [ ] Kindly update the sections with relevant information, like how to set up and run the app locally.